WikiWatershed/model-my-watershed
# Run models from compare views

The compare views are designed to be read-only in the sense that any changes to the inputs or results are not saved to the project. 

I realize the layout is a bit off -- I'll fix it in a forthcoming commit.

To test:
 * In the modeling views, change the precipitation level on a scenario.
 * Go to the compare views
 * Check that the precipitation value and results were copied over.
 * Change the precip value, and check that new results load. Make sure the spinner appears, and that you can switch to Water Quality results. While Water Quality results are visible, change the precip value and check that it hasn't switched back to the Runoff results.
 * Click back to go to the modeling views, and check that the new results and precip value were *not* copied.
 * The previous steps should work whether logged in or not.
 * Try going to the compare views as a guest and then login. The project should be saved and the URL updated, although any changes introduced by the compare views should not be saved.

Connects #593
